gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/SubscriptionServiceImpl.java

```diff
@@ -313,10 +314,14 @@ public SubscriptionEntity create(
                 throw new PlanNotYetPublishedException(plan);
             }
 
-            PlanSecurity planSecurity = genericPlanEntity.getPlanSecurity();
-            PlanSecurityType planSecurityType = PlanSecurityType.valueOfLabel(planSecurity.getType());
-            if (planSecurityType == PlanSecurityType.KEY_LESS) {
-                throw new PlanNotSubscribableException("A key_less plan is not subscribable!");
+            PlanMode planMode = genericPlanEntity.getPlanMode();
+            PlanSecurityType planSecurityType = null;
+            if (planMode == PlanMode.STANDARD) {
+                PlanSecurity planSecurity = genericPlanEntity.getPlanSecurity();
+                planSecurityType = PlanSecurityType.valueOfLabel(planSecurity.getType());
+                if (planSecurityType == PlanSecurityType.KEY_LESS) {
+                    throw new PlanNotSubscribableException("A key_less plan is not subscribable!");
```

Comment:
since we have a readable message here maybe we could remove the underscore ?
```suggestion
                    throw new PlanNotSubscribableException("A keyless plan is not subscribable!");
```